Sources/BraintreePayPal/BTPayPalVaultErrorHandlingEditRequest.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+/// Options for the PayPal edit funding instrument flow with retry on failed attempts
+///
+/// - Warning: This feature is currently in beta and may change or be removed in future releases.
+public struct BTPayPalVaultErrorHandlingEditRequest {
```

Comment:
@jaxdesmarais Wasn't this supposed to re-use the existing `BTPayPalVaultEditRequest`, and just add the `riskCorrelationID` as an optional param?

Reply:
There was a concern brought up by product where we talked about different scenarios where 1 merchant has both edit FI and error handling in different use cases. The decision was make to resist having 2 separate requests to handle that use case.

Reply:
This is still a bit confusing to me, since these two request objs input and do the exact same thing (both should allow an optional riskCorrelationID).

Do you have a paper trail of the convo for this decision?

Reply:
Got it, thanks for clarifying!

Reply:
Nit - renaming `BTPayPalVaultErrorHandlingEditRequest` --> `BTPayPalVaultEditRetryRequest`? It's unclear how the flow will be different b/w the 2 requests (though it actually won't).

Reply:
I think it's clearer with the final version of these requests merged in the feature branch where the riskCorrelationID is a required field for the error handling flow. The riskCorrelationID is just passed into the parameters function in the SDK.

Reply:
Here is the PR link where you can see the final changes: https://github.com/braintree/braintree_ios/pull/1395